tensorflow/python/compiler/tensorrt/test/tf_trt_integration_test_base.py

```diff
@@ -187,9 +187,8 @@ def BuildParams(self, graph_fn, dtype, input_shapes, output_shapes):
     """
 
     input_mask = [[False] + [True] * (len(shape) - 1) for shape in input_shapes]
-    output_mask = [
-        [False] + [True] * (len(shape) - 1) for shape in output_shapes
-    ]
+    output_mask = [[False] + [True] * (len(shape) - 1) if shape else []
```

Comment:
Can you put a comment here to explain why this line is different from the input_mask line? I think the reason is because output shape can be None while input shape can't be.